documentation/course/e2/README.md

```diff
@@ -679,38 +697,50 @@ TODO @piterek130
     <th>Wady</th>
   </tr>
   <tr>
-    <th>Rozwiązanie 1</th>
+    <th>Single Page Application (SPA)</th>
     <td>
       <ul>
-        <li>Zaleta 1</li>
+        <li>Wysoka wydajność</li>
+        <li>Płynne doświadczenie użytkownika</li>
+        <li>Kompatybilność międzyplatformowa</li>
+        <li>Zmniejszone obciążenie serwera</li>
       </ul>
     </td>
     <td>
       <ul>
-        <li>Wada 1</li>
+        <li>Początkowy czas ładowania</li>
+        <li>Problemy z SEO
```

Comment:
Seems outdated. A long time ago Googlebot was unable to run JS on crawled pages, which made it see SPAs as blank pages. However, it has actual rendering (including running JS) in its pipeline for some time now:
![googlebot-crawl-render-index](https://github.com/user-attachments/assets/a8cb3977-0ac3-49da-9e40-28992db346b6)
https://developers.google.com/search/docs/crawling-indexing/javascript/javascript-seo-basics